src/test/java/hlf/java/rest/client/IT/ChannelIT.java

```diff
@@ -24,91 +24,91 @@
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 public class ChannelIT {
 
-  @Autowired ChannelService channelService;
+    @Autowired ChannelService channelService;
```

Comment:
[nitpick] you can run `mvn fmt:check` to format the files, git captures intendation as changes in the files even if there is no actual change.